common/annotations/af.xml

```diff
@@ -811,9 +811,9 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<annotation cp="🤥" type="tts">gesig wat jok</annotation>
 		<annotation cp="🫨">aardbewing | gesig | gesig wat skud | skok | vibreer</annotation>
 		<annotation cp="🫨" type="tts">gesig wat skud</annotation>
-		<annotation cp="🙂‍↔">kop wat horisontaal skud | nee, skud kop</annotation>
```

Comment:
So the Afrikaans commas look intentional, but I also expect that since these are keywords they're probably fine.

Reply:
hm. these may need re-review next time then.

This particular comma was present in the en, though, so it may just be an intentional translation.

Reply:
Yes, I think that one is probably intentional in English and many other languages